studio/pages/new/[slug].tsx

```diff
@@ -421,6 +429,37 @@ const Wizard: NextPageWithLayout = () => {
                 )}
               </Panel.Content>
             )}
+
+            {!isSelectFreeTier && (
+              <>
+                <Panel.Content className="border-b Form section-block--body has-inputs-centered border-panel-border-interior-light dark:border-panel-border-interior-dark">
```

Comment:
> `section-block--body has-inputs-centered`

? @supabase/frontend

Reply:
@kevcodez if there's no CSS for these, worth removing? Will help to avoid confusion in the future

Reply:
Sure, I just copy & pasted from one of the Panel.Content above, will remove and doublecheck styling

Reply:
Removed all those classes in the file @alaister